LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/CommonAPI/ApiVersion.cs

```diff
@@ -254,29 +253,17 @@ public int CompareTo(ApiVersion other)
             Name == version.Name &&
             IsKnown == version.IsKnown;
 
-        public static bool operator <(ApiVersion value1, ApiVersion value2)
-        {
-            if (value1 is null) throw new ArgumentNullException(nameof(value1));
-            return value1.CompareTo(value2) < 0;
-        }
+        public static bool operator <(ApiVersion value1, ApiVersion value2) =>
+            value1 is { } v1 ? value1.CompareTo(value2) < 0 : value2 is not null;
```

Comment:
Did you mean to use `v1` here?

```suggestion
            value1 is { } v1 ? v1.CompareTo(value2) < 0 : value2 is not null;
```

Reply:
Good catch, I will inline this whole logic on the call-site though to get rid of the wrong implementation

Reply:
Addressed this in ebe9bde78806fbf0c0384f25763d9cbbfcdd14a4